components/wpt-results.html

```diff
@@ -219,7 +219,10 @@
       }
 
       _computePathIsATestFile (path) {
-        return path.endsWith('.html') || path.endsWith('.htm')
+        return path.endsWith('.html') || path.endsWith('.htm') ||
+               path.endsWith('.py') || path.endsWith('.svg') ||
+               path.endsWith('.xhtml') || path.endsWith('.xht') ||
+               path.endsWith('.xml');
```

Comment:
Thanks for the change, looks like the JS lint is failing on the `;` (we're using https://standardjs.com/)